build/configs/rtl8721csm/ipv6/defconfig

```diff
@@ -0,0 +1,1348 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# TinyAra Configuration
+#
+
+#
+# Build Setup
+#
+# CONFIG_EXPERIMENTAL is not set
+# CONFIG_DEFAULT_SMALL is not set
+CONFIG_HOST_LINUX=y
+# CONFIG_HOST_OSX is not set
+# CONFIG_HOST_WINDOWS is not set
+# CONFIG_HOST_OTHER is not set
+# CONFIG_WINDOWS_NATIVE is not set
+
+#
+# Build Configuration
+#
+CONFIG_APPS_DIR="../apps"
+CONFIG_FRAMEWORK_DIR="../framework"
+CONFIG_TOOLS_DIR="../tools"
+CONFIG_BUILD_FLAT=y
+# CONFIG_APP_BINARY_SEPARATION is not set
+# CONFIG_BUILD_2PASS is not set
+CONFIG_BOARD_BUILD_DATE=200204
```

Comment:
For this PR(https://github.com/Samsung/TizenRT/pull/5014), 
Please change to `CONFIG_BOARD_BUILD_DATE="200204"`